src/swarm/node/connection/ConnectionHandler.d

```diff
@@ -378,7 +379,7 @@ public abstract class ConnectionHandlerTemplate ( Commands : ICommandCodes )
 *******************************************************************************/
 
 abstract public class ISwarmConnectionHandler : IFiberConnectionHandlerBase,
-    Resettable, ISwarmConnectionHandlerInfo
+    Resettable, ISwarmConnectionHandlerInfo, IAddrPort
```

Comment:
Is this not the breaking change? Or does the node already provides methods `address` and `port`?